utility.py

```diff
@@ -2,7 +2,25 @@
 import logging
 import os
 import time
-logging.basicConfig(level=logging.INFO)
+import requests
+import json
+from astral import LocationInfo
+from astral.sun import sun
+import datetime
+import pytz
+
+
+def configure_logging():
+    log_level = os.getenv("LOG_LEVEL", "INFO")
+    log_format = "%(asctime)s %(levelname)-8s [%(filename)s:%(lineno)d] %(message)s"
+    log_dateformat = "%Y-%m-%d:%H:%M:%S"
+    logging.basicConfig(level=log_level, format=log_format, datefmt=log_dateformat)
+    logger = logging.getLogger()
+    logger.setLevel(level=log_level)
+    formatter = logging.Formatter(fmt=log_format, datefmt=log_dateformat )
+    handler = logger.handlers[0]
+    handler.setFormatter(formatter)
```

Comment:
Applying the format and date format to the root logger.  This automatically affects all `logging.info` etc calls and they are shown in the new format, eg

```
2021-04-03:20:48:04 INFO     [screen-weather-get.py:85] Updating SVG
```